.changes/v3.9.0/1007-bug-fixes.md

```diff
@@ -0,0 +1,3 @@
+* Added `prefix_length` field to `resourceVcdVappNetwork` to support creating IPv6 vApp networks [GH-1007]
+* Deprecated `netmask` in `resourceVcdVappNetwork` [GH-1007]
+* Removed `netmask` field's `Default` value which, if not provided before, will result in a Terraform error. The user would then need to add a `"netmask" = "255.255.255.0"` to their existing vApp networks [GH-1007] 
```

Comment:
Updated. This confused me because in go-vcloud-director (https://github.com/vmware/go-vcloud-director/blob/main/CHANGELOG.md) everything is past tense, so didn't check how it's in this project.